components/ExerciseDetails/index.js

```diff
@@ -2,40 +2,108 @@ import styled from "styled-components";
 
 export default function ExerciseDetails({ exercise }) {
   return (
-    <>
-      <figure>
+    <Container>
+      <ImageWrapper>
         <ExerciseImage src={exercise.imageUrl} />
+      </ImageWrapper>
+      <TitleAndMuscleWrapper>
+        <ExerciseTitle>{exercise.name}</ExerciseTitle>
         <ExerciseDetailsMuscle>
-          {exercise.muscleGroups.join(" ")}
+          {exercise.muscleGroups.map((muscle, index) => (
+            <MuscleGroup key={index}>{muscle}</MuscleGroup>
+          ))}
         </ExerciseDetailsMuscle>
-      </figure>
+      </TitleAndMuscleWrapper>
       <ExerciseDetailsInstruction>
         {exercise.instructions.map((instruction, index) => (
-          <li key={index}>{instruction}</li>
+          <InstructionItem key={index}>
```

Comment:
The same here. 
Better: Use the instruction itself. But be aware this may cause problems when having the same instruction twice. 
Best: Each instruction is an object that includes an id.